src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,11 +1,347 @@
+    try {
+      const { data } = await axios.post(`${baseUrl}/Users/Register`, values);
+      console.log(data);
+      toastSuccess("Account created successfully.");
+      navigate("/VerifyAccount");
+    } catch (error: any) {
+      console.log(error);
+      toastError(error?.response?.data?.message || "There's a mistake.");
+    }
+    setIsLoading(false);
+  }
+
+  function appendToFormData(values: any) {
+    const registerForm = new FormData();
+    registerForm.append("userName", values.userName);
+    registerForm.append("email", values.email);
+    registerForm.append("country", values.country);
+    registerForm.append("phoneNumber", values.phoneNumber);
+    registerForm.append("password", values.password);
+    registerForm.append("confirmPassword", values.confirmPassword);
+    registerForm.append("profileImage", values.profileImage[0]);
+    return registerForm;
+  }
+
+  
+ 
+  const onSubmit = async (values: any) => {
+    localStorage.setItem("register-Email", values?.email);
+
+    setIsLoading(true);
+    const formData = appendToFormData(values);
+    handleRegister(formData);
+  };
+
+  const togglePasswordVisibility = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const toggleConfirmPasswordVisibility = () => {
+    setShowConfirmPassword(!showConfirmPassword);
+  };
+
+
+
+function handlePic(e:any) {
+  setUserImage(URL.createObjectURL(e.target.files[0]))
+  
+  
+}
   return (
     <>
-    <div className='Register vh-100 overflow-auto pageOverflow'>
-      Register
+      <div className="auth-container py-5 p-md-0 d-flex flex-column justify-content-center align-items-center">
+        <div className="logo">
+          <img className="form-logo pb-2" src={logo} alt="logo" />
+        </div>
+        <form   
+                  onSubmit={handleSubmit(onSubmit)}
+          className="form col-md-8 col-sm-9 col-10 bg-info rounded-4 p-md-5  p-3"
+        >
+         <div className=" pt-3">
+           <p className=" text-light mb-0">welcome to PMS</p>
+           <h2 className=" form-Name mb-3">
+             <span>C</span>reate New Account
+           </h2>
+           </div>
+          
+          <div className="upload mx-auto my-3 d-flex justify-content-center align-items-center ">
+            <img className="  m-0 p-0" src={userImage||defaultImage} alt="selectImage" />
+
+            <div className="round  text-center  d-flex align-items-end  text-center justify-content-center mx-auto  ">
+              <div className="h4 carousel mx-auto text-center position-relative start-0 w-100">
+                <input
+                onChangeCapture={(e)=>handlePic(e)}
+                  {...register("profileImage")}
+                  className="m-0 p-0 position-relative"
+                  type="file"
+                />
+                <i className="fa-solid fa-camera"></i>
+              </div>
+            </div>
+          </div>
+
+          <div className="d-flex flex-wrap align-items-center    mt-2 ">
```

Comment:
We should be using `align-items:center` here, to avoid having adjacent form inputs not aligned with each other as in the case below:
![image](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/864ac678-7b26-455c-83d9-6667b5917728)